docs/config/lua/wezterm.table/index.md

```diff
@@ -0,0 +1,17 @@
+# `wezterm.table` module
```

Comment:
This file is generated and shouldn't be added.

Please add `/docs/config/lua/wezterm.table/index.md` to `.gitignore` and remove this file from the PR!

Reply:
I updated the `.gitignore` and removed that file now.